spring-boot/admin/src/main/resources/application.yml

```diff
@@ -107,3 +126,12 @@ security:
       userInfoUri: https://api.github.com/user
 
 management.security.enabled: false
+
+# Proxy configuration files are stored in this directory. The value of this parameter needs to direct to the same
+# folder as spring.cloud.config.server.native.searchLocations from the Config Servers application.yml.
+proxy.config.files.folder: ${user.dir}/.temp-gateway-config-file-storage
+
+# If CoatRack is deployed via IDE/commandline the Gateway config files will be stored in the '~/.coatrack' folder.
+# If docker-compose is used for deployment, these files will be stored internally within a docker volume by overrding
+# this parameter with 'false'.
+is-gateway-config-files-folder-in-users-home-directory: true
```

Comment:
Two comments/questions:

1. Will it actually be stored in the home folder now? Or in the folder from where you start Coatrack? In another discussion you said that "user.home" does not work in the pipeline? 
2. In any case, the parameter name is misleading, as it defines what _should_ be done rather than what _is_ the case, so I would suggest to rename as an "instruction": `store-gateway-config-files-folder-in...directory`

Reply:
Outdated.